descriptors/ModuleDescriptor-template.json

```diff
@@ -111,6 +111,15 @@
             "search.bibframe.collection.get"
           ]
         },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/search/bibframe/authorities",
+          "permissionsRequired": [
+            "search.bibframe.collection.get"
```

Comment:
was it intentional to reuse the same permission which was also for ```/search/bibframe``` or should here be new bibframe authority collection get permission?

Reply:
yes, we decided that we don't need separate permission for our module and this one can be reused

Reply:
We will create a new permission `search.bibframe.authority.collection.get`
We will also rename the existing permission to `search.bibframe.work.collection.get` as part of [MSEARCH-786](https://folio-org.atlassian.net/browse/MSEARCH-786)

Reply:
@pkjacob , is there link to some UXPROD feature (or wiki page) that describes the feature and changes being added in the scopes of MSEARCH-784 and MSEARCH-786? I am asking just to understand more about the context as I could not find more info about it and there is no relations between the stories. Looking at [MSEARCH-786](https://folio-org.atlassian.net/browse/MSEARCH-786) I am wondering maybe some better names can be suggested for the API, topic, index and permissions as from the readability point of view I did not get what ```"bibframe-work"``` could mean

Reply:
Hi @mukhiddin-yusuf , Here is the UXPROD feature -> https://folio-org.atlassian.net/browse/UXPROD-4672. The details given in the feature story is very high level. So, I am not sure if you will get the information you are looking for from this feature story.

There are lot of articles in the internet (mainly from Library of Congress) that explains Bibframe & its various components. Here are some documents.

https://www.loc.gov/bibframe/docs/bibframe2-model.html
https://en.wikipedia.org/wiki/BIBFRAME
https://www.loc.gov/bibframe/docs/bibframe2-model.html
https://www.librarianshipstudies.com/2017/12/bibframe.html


In summary,
[MSEARCH-781](https://folio-org.atlassian.net/browse/MSEARCH-781) creates an index for Bibframe Work resources 
[MSEARCH-784](https://folio-org.atlassian.net/browse/MSEARCH-784) creates an index for Bibrame Authority resources.

 [MSEARCH-786](https://folio-org.atlassian.net/browse/MSEARCH-786) is a clean up story to fix some inconsistencies in these two indexes.